kubernetes-manifests/cartservice.yaml

```diff
@@ -45,14 +45,17 @@ spec:
             fieldRef:
               fieldPath: status.hostIP
         - name: SIGNALFX_ENDPOINT_URL 
-          # value: "http://zipkin.default:9411/api/v2/spans"
           value: "http://$(NODE_IP):9411/api/v2/spans"
-        - name: SIGNALFX_SERVICE_NAME
-          value: "cartservice"
-        - name: SIGNALFX_SERVER_TIMING_CONTEXT
+        - name: SIGNALFX_LOGS_ENDPOINT_URL
+          value: "http://$(NODE_IP):4318/v1/logs"
```

Comment:
What king of logs are being sent from by the instrumentation?

Reply:
Profiling

Reply:
Just curious, is [this library](https://github.com/signalfx/signalfx-dotnet-tracing) sending profiling data in OTLP logs?

Reply:
correct

Reply:
Ok, thanks!